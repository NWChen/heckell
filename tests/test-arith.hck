let a: int;
a = 5;

let b: int;
b = 6;

let c: int;
c = a + 7;

print(c);