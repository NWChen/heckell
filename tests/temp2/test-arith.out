let a: int
a = 5
let b: int
b = 5
let c: int
c = (a + 5)
